crates/wit-parser/src/resolve.rs

```diff
@@ -144,95 +180,118 @@ impl Resolve {
                 )
             })
         } else {
-            let id = self.push_file(path)?;
-            Ok((id, vec![path.to_path_buf()]))
+            let ids = self.push_file(path)?;
+            let mut pkg_ids = Vec::new();
+            let mut path_bufs = Vec::new();
+            for id in ids {
+                pkg_ids.push(id);
+                path_bufs.push(path.to_path_buf());
+            }
+            Ok((pkg_ids, path_bufs))
         }
     }
 
+    fn _sort_unresolved_packages(
```

Comment:
It's ok in general to skip the leading underscore for private methods in Rust since the visibility (or lack thereof here) covers that. Some of the other methods have this underscore prefix but that's just because the public and private versions both wanted to be called the same thing so I gave up on naming an gave the private one a leading `_`. For this case specifically though it's ok to just call this `sort_unresolved_packages`

Reply:
Done.